smt/design_space/tests/test_design_space.py

```diff
@@ -8,17 +8,36 @@
 
 import numpy as np
 
-import smt.utils.design_space as ds
 from smt.sampling_methods import LHS
-from smt.utils.design_space import (
-    HAS_CONFIG_SPACE,
-    BaseDesignSpace,
-    CategoricalVariable,
-    DesignSpace,
-    FloatVariable,
-    IntegerVariable,
-    OrdinalVariable,
-)
+import os
+
+if os.getenv("FORCE_RUN_PLAIN_BASICS"):
```

Comment:
Would not it be the opposite? I mean when the variable is set we want to run the "else" statements? Maybe the variable should be renamed RUN_PLAIN_DESIGN_SPACE_TEST as it is only used for test purpose.